app/views/admin/whats_new/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<% content_for :page_title, t('whats_new.title') %>
+<% content_for :title, t('whats_new.title') %>
+<% content_for :title_margin_bottom, 0 %>
+
+<%
+  sections = [
+    t('whats_new.introduction'),
+    t('whats_new.recent_changes'),
+    t('whats_new.guidance'),
+  ]
+%>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <p class="govuk-body-l"><%= t('whats_new.summary') %></p>
+
+    <p class="govuk-body app-view-whats-new__last-updated"><%= t('whats_new.last_updated') %></p>
+
+    <div id="content-list">
+      <%= render "govuk_publishing_components/components/contents_list", {
+        contents: sections.map do | section |
+          {
+            href: "##{section[:heading].parameterize(separator: '-')}",
+            text: section[:heading],
+          }
+        end
+      } %>
```

Comment:
Does still match the screenshot? I'm a bit confused as I'm expecting it to loop through the recent changes to produce the screenshot - however as it maps through sections it'd only say "Recent changes"?

Reply:
Sections are the sections within the page:
```
Moving to the Design System
....

Recent changes
....

Publisher guidance, updates to GOV.UK and previous changes to Whitehall publisher
....
```

`Recent changes` section has `updates` within it:

```
Recent changes
- Update 1
- Update 2
```

Reply:
Oh sorry I misunderstood the code.